continuous_integration/setup_conda_environment.cmd

```diff
@@ -15,17 +15,14 @@ conda remove --all -q -y -n %CONDA_ENV%
 
 @rem Create test environment
 @rem (note: no cytoolz as it seems to prevent faulthandler tracebacks on crash)
-conda create -n %CONDA_ENV% -q -y python=%PYTHON% pytest toolz setuptools
+conda create -n %CONDA_ENV% -q -y python=%PYTHON%
 
 call activate %CONDA_ENV%
 
-@rem Install optional dependencies for tests
-%CONDA_INSTALL% numpy=%NUMPY% pandas=%PANDAS% 
-%CONDA_INSTALL% numba bokeh=0.12.3 toolz six mock pep8 pylint
+%CONDA% env update -f environment.yml
 
 %PIP_INSTALL% pytest-pep8
```

Comment:
@talumbau This is handled by `environment.yml`:

```
- pip:
  - pytest-pep8
```

So we can remove it here, correct?